tool/speed.cc

```diff
@@ -2627,6 +2627,7 @@ bool Speed(const std::vector<std::string> &args) {
        !SpeedECMUL(selected) ||
        // OpenSSL 1.0 doesn't support Scrypt
        !SpeedScrypt(selected) ||
+       !SpeedAESGeneric(EVP_chacha20_poly1305(), "EVP-ChaCha20-Poly1305", kTLSADLen, selected) ||
```

Comment:
Should the function name be changed because it has AES in it? Or should ChaCha20-Poly go in its own function?

Reply:
Different function name is the way to go. There's nothing specific to AES in this function.

Reply:
perhaps `SpeedAEADGeneric`?